website/src/index.html

```diff
@@ -0,0 +1,210 @@
+    <meta name="description" content="Log work hours and get notifies when it's time to leave" />
+    <link rel="apple-touch-icon" sizes="180x180" href="./images/resources/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/resources/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./images/resources/favicon-16x16.png">
+    <link rel="manifest" href="./images/resources/site.webmanifest">
+    <link rel="mask-icon" href="./images/resources/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="shortcut icon" href="./images/resources/favicon.ico">
+    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-config" content="./images/resources/browserconfig.xml">
+    <meta name="theme-color" content="#9a93c6">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div id="loader" class="loader"></div>
+        <nav id="navbar" class="header__nav">
+            <ul>
+                <li class=" header__nav__logo">
+                    <a href="#">
+                        <img src="images/resources/timer.svg" alt="Time To Leave Logo">
+                    </a>
+                </li>
+                <li class="header__nav__item header__nav__button">
+                    <a href="https://github.com/thamara/time-to-leave"><img src="./images/github.svg" alt="Github logo"></a>
+                </li>
+                <!--<li class="header__nav__item header__nav__button">
+                    <a href="#"><img src="./images/switch.svg" alt=""></a>
+                </li>-->
+                <li class="header__nav__item header__nav__button">
+                    <a href="https://github.com/thamara/time-to-leave/releases/tag/v.1.5.5" class="button--primary">Download</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+
+    <main id="app">
+        <section class="intro">
+            <div class="marketing">
+                <div class="marketing__container js-reveal">
+                    <h1 class="intro__title">
+                        Time to Leave
+                        <img src="images/resources/timer.svg" alt="Time To Leave Logo">
+                    </h1>
+
+                    <p class="marketing__desc">
+                        Time to Leave is a Desktop app for work time management, looking similar to a time-card system.
+                        It’s
+                        different from other software available as it’s very simple, doesn’t require any database
+                        configuration, and focuses on allowing you to control your time. Its premise is to help you
+                        manage
+                        the time you are working and to notify you when it’s time to leave (therefore the name).
```

Comment:
```suggestion
                        It’s different from other software available as it’s very simple, doesn’t require any database
                        configuration, and focuses on allowing you to control your time. Its premise is to help you
                        manage the time you are working and to notify you when it’s time to leave (therefore the name).
```
I think the line breaks were weird here